__tests__/integration/utxo-filter.test.js

```diff
@@ -271,28 +274,27 @@ describe('utxo-filter routes', () => {
 
     // Validating filtered response
     expect(utxosObj.utxos).toHaveProperty('length', 2);
-    expect(utxosObj.total_amount_available).toBe(30);
     expect(utxosObj.total_utxos_available).toBe(2);
     expect(utxosObj.total_amount_locked).toBe(0);
     expect(utxosObj.total_utxos_locked).toBe(0);
 
     // Validating unfiltered response
     const unfilteredObj = unfilteredResponse.body;
     expect(unfilteredObj.total_utxos_available).toBeGreaterThan(utxosObj.total_utxos_available);
-    for (const utxoIndex in utxosObj.utxos) {
-      // We expect the results to be in the same order
-      expect(unfilteredObj.utxos[utxoIndex]).toStrictEqual(utxosObj.utxos[utxoIndex]);
-    }
-
-    const utxo0 = utxosObj.utxos[0];
-    expect(utxo0.address).toBe(transactions.tx10.address);
-    expect(utxo0.amount).toBe(10);
-    expect(utxo0.locked).toBe(false);
-
-    const utxo1 = utxosObj.utxos[1];
-    expect(utxo1.address).toBe(transactions.tx20.address);
-    expect(utxo1.amount).toBe(20);
-    expect(utxo1.locked).toBe(false);
+    // for (const utxoIndex in utxosObj.utxos) {
```

Comment:
Why did you remove this validation?

Reply:
This has since changed, but we cannot guarantee the order of the utxos now.
The main reason is that since we are getting the most recent utxos first, the first 2 available utxos on the filter would be the one from tx50 and the change from it, but we shuffle when we have a change so its position will not be fixed.